lib/internal/modules/helpers.js

```diff
@@ -63,6 +63,18 @@ function toRealPath(requestPath) {
   });
 }
 
+/** @type {Set<string>} */
+let userConditions;
+
+function getUserConditions() {
+  if (userConditions === undefined) {
+    userConditions = new SafeSet([
```

Comment:
We should never expose SafeSet to the user. That would make it unsafe.

Reply:
Ah, I forgot I have to run the lint separately.
Should I add a ignore comment? or should I use an array instead?
```
/home/runner/work/node/node/lib/internal/modules/helpers.js
Error:   71:26  error  Use `const { SafeSet } = primordials;` instead of the global  node-core/prefer-primordials
```

Reply:
I'm not sure what should be done. @nodejs/linting

Reply:
`SafeSet` should have worked, I think. Let me poke around a bit....

Reply:
`SafeSet` works for me locally. I'll add a suggestion to the current version of this file (which is still failing linting).

Reply:
Oh...shouldn't expose it to users. Right. OK. Um....let me look more at this....

Reply:
Looks like Jacob has a suggestion below and I'm just making a lot of noise. Sorry about that.